public/components/overview/components/dashboard_controls.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSuperDatePicker,
+  EuiText,
+} from '@elastic/eui';
+import React from 'react';
+import { OnTimeChangeProps } from '@opensearch-project/oui/src/eui_components/date_picker/super_date_picker/super_date_picker';
+import { coreRefs } from '../../../framework/core_refs';
+import { HOME_CONTENT_AREAS } from '../../../plugin_helpers/plugin_overview';
+
+interface Props {
+  dashboardTitle: string;
+  startDate: string;
+  endDate: string;
+  setStartDate: (start: string) => void;
+  setEndDate: (end: string) => void;
+  showModal: () => void;
+}
+
+export function DashboardControls(props: Props) {
+  const onTimeChange = (onTimeChangeProps: OnTimeChangeProps) => {
+    props.setStartDate(onTimeChangeProps.start);
+    props.setEndDate(onTimeChangeProps.end);
+
+    coreRefs.contentManagement?.updatePageSection(HOME_CONTENT_AREAS.DASHBOARD, (section) => {
```

Comment:
Allows the date-picker to update the displayed dashboard. Pending https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7651